source/features/parse-backticks.tsx

```diff
@@ -11,21 +11,24 @@ function init(): void {
 		'.commit-desc', // `isCommit`, `isCommitList`, `isRepoTree`
 		'.commit-message', // Pushed commits in `isPRConversation`, `isCompare`, `isReleasesOrTags`
 		'.message', // `isCommitList`, `isRepoTree`, `isBlame`
-		'.Box--condensed .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container .link-gray[href*="/commit/"]', // `isSingleFile`
+		'.repository-content .js-details-container pre', // `isSingleFile`
```

Comment:
These two are commit title and description:
![image](https://user-images.githubusercontent.com/44045911/86459421-54c29a00-bd59-11ea-90a1-01760ef8e57c.png)